modules/new_profile/php/new_profile.class.inc

```diff
@@ -42,68 +44,6 @@ class New_Profile extends \NDB_Form
 
         return false;
     }
-    /**
```

Comment:
if you moved this to the ajax script, you can actually keep the code here in process(). see https://github.com/aces/Loris/pull/3924 for example. In the handleSubmit() of the jsx index file, i appended 'fire_away' to the formObject and named the submit button 'fire_away'. this allows for the process function to be accessed

Reply:
It doesn't work for me in this case, I think.
I did the same way as media module.